### PR TITLE
Add issue template with list of repos

### PR DESCRIPTION
**Pull request by natemcmaster**
We get lots of noise here, but no one is really following up on issues. This message will appear in the issue box when users try to make a new issue. Hopefully this will help them find the right place to file issues.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+This repo is not closely monitored by product and development teams.
+If you are experiencing a product issue, please open the issue on one of these repositories instead.
+
+Common ASP.NET repos and projects:
```
Comment (Eilon) [inline .github/ISSUE_TEMPLATE.md]: ... and Entity Framework!

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+ - Documentation: https://github.com/aspnet/Docs
+ - MVC: https://github.com/aspnet/Mvc
+ - Entity Framework Core: https://github.com/aspnet/EntityFramework
+ - SignalR: https://github.com/aspnet/SignalR
+ - ASP.NET Core in Docker: https://github.com/aspnet/aspnet-docker
+ - Microsoft.NET.Sdk.Web: https://github.com/aspnet/websdk
+ - Kestrel HTTP Server: https://github.com/aspnet/KestrelHttpServer
+
+Common projects that are not ASP.NET:
```
Comment (Eilon) [inline .github/ISSUE_TEMPLATE.md]: I'd just say "Other common projects"?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+ - Visual Studio: https://developercommunity.visualstudio.com
+ - NuGet: https://github.com/NuGet/home
+ - .NET Core CLI, SDK, or runtime: https://github.com/dotnet/core
```
Comment (Eilon) [inline .github/ISSUE_TEMPLATE.md]: Is that repo for all of those projects? CLI is in https://github.com/dotnet/cli, no?

**Review by Eilon — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (natemcmaster) [inline .github/ISSUE_TEMPLATE.md]: dotnet/core is the .NET Core "Home" repo. It contains a list of relevant repos including dotnet/cli, dotnet/sdk, and others.

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,20 +1,23 @@
...
  - Documentation: https://github.com/aspnet/Docs
  - MVC: https://github.com/aspnet/Mvc
  - Entity Framework Core: https://github.com/aspnet/EntityFramework
+ - Entity Framework 6: https://github.com/aspnet/EntityFramework6
  - SignalR: https://github.com/aspnet/SignalR
  - ASP.NET Core in Docker: https://github.com/aspnet/aspnet-docker
  - Microsoft.NET.Sdk.Web: https://github.com/aspnet/websdk
  - Kestrel HTTP Server: https://github.com/aspnet/KestrelHttpServer
+ - ASP.NET Katana: https://github.com/aspnet/AspNetKatana
+ - and more! See a full list here: https://github.com/aspnet
```
Comment (Eilon) [inline .github/ISSUE_TEMPLATE.md]: Nice 😄

**Review by Eilon — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+This repo is not closely monitored by product and development teams.
```
Comment (Eilon) [inline .github/ISSUE_TEMPLATE.md]: Oh actually wait, one more thing. The point isn't about whether it's monitored or not, but rather about how to get issues resolved most quickly. I think this text might represent that better:

> To address your issues more quickly, please open the issue in the most appropriate repository. These are some of the most common repos:

**Review by Eilon — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 5 above)
Comment (natemcmaster) [inline .github/ISSUE_TEMPLATE.md]: updated

**Review by Eilon — APPROVED** [review]
:shipit: 😄

**Pull request merged**